openarm_hardware/CMakeLists.txt

```diff
@@ -72,6 +73,7 @@ if(BUILD_TESTING)
   )
   ament_target_dependencies(test_openarm_hardware
```

Comment:
It seems that `ament_target_dependencies()` is deprecated: https://github.com/ament/ament_cmake/pull/572

Could you `target_link_libraries()` instead?